.github/workflows/nightly-jdk8-scala2.13.yml

```diff
@@ -0,0 +1,28 @@
+name: Nightly cross-validate and test
+
+on:
+  schedule:
+    - cron:  '0 4,21 * * *'
+
+jobs:
+  compile-and-test:
+    name: Compile and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
```

Comment:
we can evaluate to use an action that supports matrix build (e.g.: https://github.com/lightbend/cloudflow/blob/13c0cd651453bafeeb5709eef060212cbce6926a/.github/workflows/build-pr.yaml#L17 ) to avoid those duplications

Reply:
that was my first approach and then realized it's not what these jobs are doing. Each job is different in what sbt commands it runs and which JVM arguments it uses.

I noticed when I prepared the build matrix and the JDK11 jobs failed because some JVM argument didn't exist (only exists on JDK8).

For the time being I'm only porting as literally as possible what we had in Jenkins to GHActions.